lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -225,6 +225,60 @@ describe('GatherRunner', function() {
     });
   });
 
+  it('clears the network cache and origin storage', () => {
+    const asyncFunc = () => Promise.resolve();
+    const tests = {
+      calledDisableNetworkCache: false,
+      calledClearStorage: false,
+    };
+    const createCheck = variable => () => {
```

Comment:
it would be nice to replace fake-driver with a nice interface around something like this :)